packages/blade/src/components/Dropdown/useDropdown.ts

```diff
@@ -177,6 +190,12 @@ const useDropdown = (): UseDropdownReturnValue => {
    * Blur handler on combobox. Also handles the selection logic when user moves focus
    */
   const onSelectBlur = (): void => {
+    if (rest.hasFooterAction) {
+      // When Footer has action buttons, we ignore the blur (by setting shouldIgnoreBlur to true in onSelectKeyDown)
```

Comment:
ignore blur on dropdown or select?

Reply:
on select.

Which is why it also has shouldIgnoreBlur. We ignore the blur when click is on dropdown item. This function itself is called on select blur.

Reply:
Offtopic: I am renaming things from `onSelectBlur` to `onTriggerBlur` since there can be confusion that it is attached to select input (it is not)